cql-parser/src/main/scala/troy/cql/parser/CqlParser.scala

```diff
@@ -38,7 +38,7 @@ object CqlParser extends JavaTokenParsers
     parse(phrase(selectStatement <~ semicolon.?), input)
 
   def parseDML(input: String): ParseResult[DataManipulation] =
-    parse(phrase(dmlDefinition <~ semicolon.?), input)
+    parse(phrase(dmlDefinition <~ semicolon.? <~ comment.?), input)
```

Comment:
Comments wont't be supported inside queries. Instead they should be supported inside schema.cql file only.